apps/apps.c

```diff
@@ -79,7 +79,7 @@ int chopup_args(ARGS *arg, char *buf)
 
     for (p = buf;;) {
         /* Skip whitespace. */
-        while (*p && isspace(_UC(*p)))
+        while (*p && ossl_isspace(_UC(*p)))
```

Comment:
For example here it's definitely not really appropriate to replace isspace. Because we are looking at user supplied string that was *composed* in current locale and we are supposed to "embrace" it. While ossl_isspace is primarily about rejecting something that can be harmful, especially to cryptographically significant operation.